From c2f8d335168ab9c57aeb7f4d64c5ffb5581eb074 Mon Sep 17 00:00:00 2001
From: imide <imide@imide.xyz>
Date: Tue, 28 Jul 2026 10:14:36 -0600
Subject: [PATCH] fix(meta): purge bugged release in metainfo

right now it only shows the most recent release which is fine.
---
 app.legcord.Legcord.metainfo.xml | 250 --------------------------
 1 file changed, 250 deletions(-)

diff --git a/app.legcord.Legcord.metainfo.xml b/app.legcord.Legcord.metainfo.xml
index 3bcbe55d..8f00c77c 100644
--- a/app.legcord.Legcord.metainfo.xml
+++ b/app.legcord.Legcord.metainfo.xml
@@ -75,256 +75,6 @@
         </ul>
       </description>
     </release>
-    <release version="1.2.4" date="2026-04-15" type="stable">
-      <url>https://github.com/Legcord/Legcord/releases/tag/v1.2.4</url>
-      <description>
-        <p>What’s New in Legcord v1.2.4 ✨</p>
-        <ul>
-          <li>🐧 **Huge streaming performance improvements on Linux**</li>
-        </ul>
-        <ul>
-          <li>🔄 **Separate “No Bundle Updates” setting for each client mod**</li>
-        </ul>
-        <ul>
-          <li>💾 **Backup &amp; Restore added**</li>
-        </ul>
-        <ul>
-          <li>🎮 **Improved macOS game detection**</li>
-        </ul>
-        <ul>
-          <li>🛡️ **New CSP options setting**</li>
-        </ul>
-        <ul>
-          <li>🪟 **Window material setting added** for supported systems ([https://github.com/Legcord/Legcord/pull/1025](https://github.com/Legcord/Legcord/pull/1025) by @passionvine).</li>
-          <li>📦 **OPTIONAL automatic Linux updates added** ([https://github.com/Legcord/Legcord/pull/1027](https://github.com/Legcord/Legcord/pull/1027) by @KarboXXX).</li>
-          <li>🎛️ **Support banner popup added**</li>
-        </ul>
-        <ul>
-          <li>🖥️ Fixed Linux `StartupWMClass` to `legcord` ([https://github.com/Legcord/Legcord/pull/1021](https://github.com/Legcord/Legcord/pull/1021) by @khancyr).</li>
-          <li>📺 Fixed screensharing quality, FPS, and audio ([https://github.com/Legcord/Legcord/pull/1026](https://github.com/Legcord/Legcord/pull/1026) by @KarboXXX).</li>
-          <li>🎚️ Fixed scrollbar styling on file preview ([https://github.com/Legcord/Legcord/pull/1024](https://github.com/Legcord/Legcord/pull/1024) by @MahmodZE).</li>
-          <li>🔊 Fixed mute toggle sound effect playback ([https://github.com/Legcord/Legcord/pull/1031](https://github.com/Legcord/Legcord/pull/1031) by @psw01).</li>
-          <li>🎮 Fixed RPC status not disappearing properly ([https://github.com/Legcord/Legcord/pull/1032](https://github.com/Legcord/Legcord/pull/1032) by @Adrigamer2950).</li>
-          <li>🔐 Fixed security prompt appearing with no passkey ([https://github.com/Legcord/Legcord/pull/1033](https://github.com/Legcord/Legcord/pull/1033) by @dequeues).</li>
-          <li>🇫🇷 Added missing French translations for settings categories ([https://github.com/Legcord/Legcord/pull/1028](https://github.com/Legcord/Legcord/pull/1028) by @youtsuhodev).</li>
-          <li>⌨️ Polished keybinds UI ([https://github.com/Legcord/Legcord/pull/1038](https://github.com/Legcord/Legcord/pull/1038) by @youtsuhodev).</li>
-          <li>🎥 Fixed camera/mic device switching not working ([https://github.com/Legcord/Legcord/pull/1051](https://github.com/Legcord/Legcord/pull/1051) by @narasaka).</li>
-          <li>⚙️ Fixed settings shortcut behavior ([https://github.com/Legcord/Legcord/pull/1046](https://github.com/Legcord/Legcord/pull/1046) by @KirobotDev).</li>
-          <li>📏 Fixed dropdown height limit in Games settings ([https://github.com/Legcord/Legcord/pull/1045](https://github.com/Legcord/Legcord/pull/1045) by @youtsuhodev).</li>
-          <li>📐 Fixed dropdown max height calculation ([https://github.com/Legcord/Legcord/pull/1059](https://github.com/Legcord/Legcord/pull/1059) by @MahmodZE).</li>
-          <li>🧹 Cleanup of old fix-me notes and code polish ([https://github.com/Legcord/Legcord/pull/1060](https://github.com/Legcord/Legcord/pull/1060) by @KirobotDev).</li>
-          <li>📂 Fixed restore zip path traversal issue ([https://github.com/Legcord/Legcord/pull/1061](https://github.com/Legcord/Legcord/pull/1061) by @youtsuhodev).</li>
-          <li>🛡️ Added stricter CSP, better permission handling, and mod validation ([https://github.com/Legcord/Legcord/pull/1069](https://github.com/Legcord/Legcord/pull/1069) by @KirobotDev).</li>
-          <li>Fixed first time setup</li>
-        </ul>
-      </description>
-    </release>
-    <release version="1.2.3" date="2026-04-15" type="stable">
-      <url>https://github.com/Legcord/Legcord/releases/tag/v1.2.3</url>
-      <description>
-        <p>What’s New in Legcord v1.2.3 ✨</p>
-        <ul>
-          <li>🐧 **Huge streaming performance improvements on Linux**</li>
-        </ul>
-        <ul>
-          <li>🔄 **Separate “No Bundle Updates” setting for each client mod**</li>
-        </ul>
-        <ul>
-          <li>💾 **Backup &amp; Restore added**</li>
-        </ul>
-        <ul>
-          <li>🎮 **Improved macOS game detection**</li>
-        </ul>
-        <ul>
-          <li>🛡️ **New CSP options setting**</li>
-        </ul>
-        <ul>
-          <li>🪟 **Window material setting added** for supported systems ([https://github.com/Legcord/Legcord/pull/1025](https://github.com/Legcord/Legcord/pull/1025) by @passionvine).</li>
-          <li>📦 **OPTIONAL automatic Linux updates added** ([https://github.com/Legcord/Legcord/pull/1027](https://github.com/Legcord/Legcord/pull/1027) by @KarboXXX).</li>
-          <li>🎛️ **Support banner popup added**</li>
-        </ul>
-        <ul>
-          <li>🖥️ Fixed Linux `StartupWMClass` to `legcord` ([https://github.com/Legcord/Legcord/pull/1021](https://github.com/Legcord/Legcord/pull/1021) by @khancyr).</li>
-          <li>📺 Fixed screensharing quality, FPS, and audio ([https://github.com/Legcord/Legcord/pull/1026](https://github.com/Legcord/Legcord/pull/1026) by @KarboXXX).</li>
-          <li>🎚️ Fixed scrollbar styling on file preview ([https://github.com/Legcord/Legcord/pull/1024](https://github.com/Legcord/Legcord/pull/1024) by @MahmodZE).</li>
-          <li>🔊 Fixed mute toggle sound effect playback ([https://github.com/Legcord/Legcord/pull/1031](https://github.com/Legcord/Legcord/pull/1031) by @psw01).</li>
-          <li>🎮 Fixed RPC status not disappearing properly ([https://github.com/Legcord/Legcord/pull/1032](https://github.com/Legcord/Legcord/pull/1032) by @Adrigamer2950).</li>
-          <li>🔐 Fixed security prompt appearing with no passkey ([https://github.com/Legcord/Legcord/pull/1033](https://github.com/Legcord/Legcord/pull/1033) by @dequeues).</li>
-          <li>🇫🇷 Added missing French translations for settings categories ([https://github.com/Legcord/Legcord/pull/1028](https://github.com/Legcord/Legcord/pull/1028) by @youtsuhodev).</li>
-          <li>⌨️ Polished keybinds UI ([https://github.com/Legcord/Legcord/pull/1038](https://github.com/Legcord/Legcord/pull/1038) by @youtsuhodev).</li>
-          <li>🎥 Fixed camera/mic device switching not working ([https://github.com/Legcord/Legcord/pull/1051](https://github.com/Legcord/Legcord/pull/1051) by @narasaka).</li>
-          <li>⚙️ Fixed settings shortcut behavior ([https://github.com/Legcord/Legcord/pull/1046](https://github.com/Legcord/Legcord/pull/1046) by @KirobotDev).</li>
-          <li>📏 Fixed dropdown height limit in Games settings ([https://github.com/Legcord/Legcord/pull/1045](https://github.com/Legcord/Legcord/pull/1045) by @youtsuhodev).</li>
-          <li>📐 Fixed dropdown max height calculation ([https://github.com/Legcord/Legcord/pull/1059](https://github.com/Legcord/Legcord/pull/1059) by @MahmodZE).</li>
-          <li>🧹 Cleanup of old fix-me notes and code polish ([https://github.com/Legcord/Legcord/pull/1060](https://github.com/Legcord/Legcord/pull/1060) by @KirobotDev).</li>
-          <li>📂 Fixed restore zip path traversal issue ([https://github.com/Legcord/Legcord/pull/1061](https://github.com/Legcord/Legcord/pull/1061) by @youtsuhodev).</li>
-          <li>🛡️ Added stricter CSP, better permission handling, and mod validation ([https://github.com/Legcord/Legcord/pull/1069](https://github.com/Legcord/Legcord/pull/1069) by @KirobotDev).</li>
-        </ul>
-      </description>
-    </release>
-    <release version="1.2.2" date="2026-02-21" type="stable">
-      <url>https://github.com/Legcord/Legcord/releases/tag/v1.2.2</url>
-      <description>
-        <p>What’s New in Legcord v1.2.2 ✨</p>
-        <ul>
-          <li>🚫 **Game blacklist for arRPC detection**</li>
-        </ul>
-        <ul>
-          <li>⚡ **Electron updated** for better stability and compatibility.</li>
-          <li>🎨 **Quick CSS toggle added**.</li>
-          <li>🪟 Fixed **Windows titlebar duplication** ([https://github.com/Legcord/Legcord/pull/1009](https://github.com/Legcord/Legcord/pull/1009) by @passionvine).</li>
-          <li>🧪 **Custom Chromium flags support**</li>
-        </ul>
-        <ul>
-          <li>🛠️ Fixed **settings menu version patch** ([https://github.com/Legcord/Legcord/pull/1014](https://github.com/Legcord/Legcord/pull/1014) by @MahmodZE).</li>
-          <li>🧹 Fixed various **build warnings** ([https://github.com/Legcord/Legcord/pull/1017](https://github.com/Legcord/Legcord/pull/1017) by @MahmodZE).</li>
-        </ul>
-      </description>
-    </release>
-    <release version="1.2.1" date="2026-01-30" type="stable">
-      <url>https://github.com/Legcord/Legcord/releases/tag/v1.2.1</url>
-      <description>
-        <p>What’s New in Legcord v1.2.1 ✨</p>
-        <ul>
-          <li>🧯 Fixed screenshare crashing when opening it up</li>
-        </ul>
-      </description>
-    </release>
-    <release version="1.2.0" date="2026-01-29" type="stable">
-      <url>https://github.com/Legcord/Legcord/releases/tag/v1.2.0</url>
-      <description>
-        <p>What’s New in Legcord v1.2.0 ✨</p>
-        <ul>
-          <li>🎮 **Brand-new game detection system**</li>
-        </ul>
-        <ul>
-          <li>🧪 **Legcord Games (Beta)**</li>
-        </ul>
-        <ul>
-          <li>🎨 **Improved Vencord Quick CSS patch**</li>
-        </ul>
-        <ul>
-          <li>🌍 **Localization improvements**</li>
-        </ul>
-        <ul>
-          <li>📺 **Stream quality patch updated**</li>
-        </ul>
-        <ul>
-          <li>⬇️ Fixed the **“Download Apps” button** incorrectly appearing.</li>
-          <li>🪟 **macOS &amp; Windows titlebar styling fixes**.</li>
-          <li>🧩 **Extended plugin abilities (optional)**</li>
-        </ul>
-        <ul>
-          <li>🎨 Numerous **style updates** to match Discord’s settings revamp and rebrand.</li>
-          <li>🎛️ Improved **Touch Bar handling** on macOS.</li>
-          <li>⚡ **Settings are now cached**, improving overall performance.</li>
-          <li>🔗 **Discord deeplinks support** (`discord://`)</li>
-        </ul>
-        <ul>
-          <li>🧯 Fixed arRPC crashing ([https://github.com/Legcord/Legcord/pull/949](https://github.com/Legcord/Legcord/pull/949) by @PONYMODZ)</li>
-          <li>▶️ Fixed YouTube adblock issues ([https://github.com/Legcord/Legcord/pull/979](https://github.com/Legcord/Legcord/pull/979) by @Alangopro)</li>
-          <li>📖 Updated README with Debian installation instructions ([https://github.com/Legcord/Legcord/pull/972](https://github.com/Legcord/Legcord/pull/972) by @khancyr)</li>
-          <li>🧭 Fixed Equicord headerbar button overlap ([https://github.com/Legcord/Legcord/pull/990](https://github.com/Legcord/Legcord/pull/990) by @lbrooney)</li>
-          <li>🎨 Updated CSS variable names used on navigation buttons ([https://github.com/Legcord/Legcord/pull/989](https://github.com/Legcord/Legcord/pull/989) by @lbrooney)</li>
-          <li>📐 Fixed trailing bar collision with the navigation bar ([https://github.com/Legcord/Legcord/pull/993](https://github.com/Legcord/Legcord/pull/993) by @lbrooney)</li>
-          <li>📺 Fixed YouTube embeds in chat and Watch Together activity ([https://github.com/Legcord/Legcord/pull/994](https://github.com/Legcord/Legcord/pull/994) by @MahmodZE)</li>
-          <li>🧩 Refactored and redesigned several UI components ([https://github.com/Legcord/Legcord/pull/995](https://github.com/Legcord/Legcord/pull/995) by @MahmodZE)</li>
-          <li>⌨️ Improved the keybind maker ([https://github.com/Legcord/Legcord/pull/996](https://github.com/Legcord/Legcord/pull/996) by @MahmodZE)</li>
-          <li>🚀 Fixed a launch failure related to global shortcuts ([https://github.com/Legcord/Legcord/pull/1004](https://github.com/Legcord/Legcord/pull/1004) by @MahmodZE)</li>
-          <li>🖥️ Fixed an issue where the screenshare picker modal closes on click-outside and wouldn’t reopen ([https://github.com/Legcord/Legcord/pull/1002](https://github.com/Legcord/Legcord/pull/1002) by @MahmodZE)</li>
-        </ul>
-      </description>
-    </release>
-    <release version="v1.1.3" date="2025-05-04" type="stable">
-      <url>https://github.com/Legcord/Legcord/releases/tag/v1.1.3</url>
-      <description>
-        <p>Improvements &amp; Fixes</p>
-        <ul>
-          <li>🔧 **Fixed the app showing a blank screen on some platforms**.</li>
-        </ul>
-        <p>Special Thanks ❤️</p>
-      </description>
-    </release>
-    <release version="v1.1.2" date="2025-05-04" type="stable">
-      <url>https://github.com/Legcord/Legcord/releases/tag/v1.1.2</url>
-      <description>
-        <p>Improvements &amp; Fixes</p>
-        <ul>
-          <li>🌈 **Full Discord rebrand support - proper styling of custom/overlay titlebars**</li>
-          <li>🚀 **Improved how Blink feature flags are set — now done only once for better reliability.**.</li>
-        </ul>
-        <p>New Contributors 👋</p>
-        <p>Special Thanks ❤️</p>
-      </description>
-    </release>
-    <release version="v1.1.1" date="2025-03-16" type="stable">
-      <url>https://github.com/Legcord/Legcord/releases/tag/v1.1.1</url>
-      <description>
-        <p>Improvements &amp; Fixes</p>
-        <ul>
-          <li>🛠️ **arRPC now runs in a separate process**, resolving #808</li>
-          <li>🌈 **Added modern macOS transparency support**.</li>
-          <li>🚀 **Upgraded Electron**, fixing #839</li>
-          <li>📦 Moved <code>tsx</code> to <code>devDependencies</code> #835</li>
-          <li>🎛️ Fixed traffic lights not being centered in "overlay" mode #860</li>
-          <li>📖 Added MacPorts installation instructions to the README #834</li>
-        </ul>
-        <p>New Contributors 👋</p>
-        <ul>
-          <li>@akierig - #834</li>
-          <li>@fabricionaweb - #835</li>
-          <li>@nihaals - #860</li>
-        </ul>
-        <p>Special Thanks ❤️</p>
-      </description>
-    </release>
-    <release version="v1.1.0" date="2025-01-13" type="stable">
-      <url>https://github.com/Legcord/Legcord/releases/tag/v1.1.0</url>
-      <description>
-        <ul>
-          <li>**Theme Manager v2:**  </li>
-          <li>**Screenshare v3 (rewrite of the rewrite):**  </li>
-          <li>**macOS Touch Bar integration:**  </li>
-          <li>**New built-in Quick CSS Editor**</li>
-          <li>**New first-time setup by @FlintSH**  </li>
-          <li>**New power management features**  </li>
-          <li>**New settings:**</li>
-          <li>**Rewritten preload to be sandboxed**</li>
-        </ul>
-      </description>
-    </release>
-    <release version="v1.0.8" date="2025-01-21" type="stable">
-      <url>https://github.com/Legcord/Legcord/releases/tag/v1.0.8</url>
-      <description/>
-    </release>
-    <release version="v1.0.7" date="2025-01-20" type="stable">
-      <url>https://github.com/Legcord/Legcord/releases/tag/v1.0.7</url>
-      <description/>
-    </release>
-    <release version="v1.0.6" date="2024-12-30" type="stable">
-      <url>https://github.com/Legcord/Legcord/releases/tag/v1.0.6</url>
-      <description/>
-    </release>
-    <release version="v1.0.5" date="2024-11-26" type="stable">
-      <url>https://github.com/Legcord/Legcord/releases/tag/v1.0.5</url>
-      <description/>
-    </release>
-    <release version="v1.0.4" date="2024-11-15" type="stable">
-      <url>https://github.com/Legcord/Legcord/releases/tag/v1.0.4</url>
-      <description/>
-    </release>
-    <release version="v1.0.3" date="2024-11-12" type="stable">
-      <url>https://github.com/Legcord/Legcord/releases/tag/v1.0.3</url>
-      <description/>
-    </release>
-    <release version="v1.0.2" date="2024-11-19" type="stable">
-      <url>https://github.com/Legcord/Legcord/releases/tag/v1.0.2</url>
-      <description/>
-    </release>
-    <release version="v1.0.0" date="2024-10-11" type="stable">
-      <url>https://github.com/Legcord/Legcord/releases/tag/v1.0.0</url>
-      <description/>
-    </release>
   </releases>
   <url type="homepage">https://legcord.app</url>
   <url type="bugtracker">https://github.com/Legcord/Legcord/issues</url>
